packages/babel-helper-define-map/src/index.js

```diff
@@ -65,8 +65,8 @@ export function push(mutatorMap: Object, node: Object, kind: string, file, scope
     value = nameFunction({ id: key, node: value, scope });
 
     // Class methods don't have their name bound in the funciton body.
-    if (t.isClassMethod(node)) {
-      value.id[t.NOT_LOCAL_BINDING] = true
+    if (t.isClassMethod(node) && value.id) {
```

Comment:
This is definitely better since it won't run without it but you will probably need to set `NOT_LOCAL_BINDING` on the `FunctionExpression` which is inside the `CallExpression`. I'm just guessing though so I'm probably wrong.